### DISCUSSION TITLE
Unable to find .NET SDK and resolve for net8.0-windows7.0

### DISCUSSION THREAD

**Discussion by notandrei21** [winstxnhdw/lc-hax#426] [Q&A] [answered]
okay, I followed the installation process without any problems until I run `requirements` 
I already have `8.0 SDK (v8.0.201)`, even after uninstalling and repeating I still run into this problem

![image](https://github.com/winstxnhdw/lc-hax/assets/158371329/006f093a-2c38-4977-b26e-571c379c86fb)

**Comment by winstxnhdw** [comment]
Restart your computer.

**Reply by notandrei21** [reply]
still occurring

**Reply by winstxnhdw** [reply]
https://github.com/dotnet/core/issues/5962#issuecomment-780084535

**Comment by notandrei21** [comment]
I guess it worked but now this happens 

![image](https://github.com/winstxnhdw/lc-hax/assets/158371329/203daac1-d23c-4fdd-8a22-4af45b97af91)

**Reply by winstxnhdw** [reply]
One other dude got this error as well. For some reason, .NET thinks you are on Windows 7. I have no idea how to fix this yet. In the meantime, you can try another injector.

**Reply by winstxnhdw** [reply]
You can also try removing this line in your folder: https://github.com/winstxnhdw/SharpMonoInjectorCore/blob/main/SharpMonoInjector.csproj#L9

**Reply by winstxnhdw** [reply, marked as answer]
FWIW, you can also try clearing your NuGet cache: `dotnet nuget locals all --clear`

**Reply by notandrei21** [reply]
I've tried both methods and it still did not work, however when I deleted NuGet from `AppData\Roaming\NuGet` and then restoring it, the errors seemed to have fixed itself and the launch process was a success

**Reply by winstxnhdw** [reply]
Strange but I am glad that it worked.
